### PR TITLE
In reference to issue #126 search option enhancement

### PR DESCRIPTION
**Pull request by nikhilrayaprolu**
In reference to issue #126,
started working on advanced component, since it is a large issue with multiple small tasks that need to be done, different implementations are required for different inputs, solving this issue iteratively.
this is the pull with iteration 1:-
image

google's advanced search:-

image
demo link available at https://susper-pr-255.herokuapp.com/

### CODE REVIEW

**Comment by nikhilrayaprolu** [comment]
implemented the first part of #126 will continue do the 2nd and 3rd on next pulls.

**Comment by mariobehling** [comment]
Please avoid new colors and keep it simple. Simply follow Google in design for now, please.

**Comment by nikhilrayaprolu** [comment]
I have updated the picture to show google's advanced search
sure I will check with the alignments.
secondly, @harshit98 you were complaining about writing the component in each file. I didn't face that problem you could check the test files I updated.

**Comment by harshit98** [comment]
@nikhilrayaprolu Yes I saw that. I'll follow the same next time. Good work!

**Comment by harshit98** [comment]
@nikhilrayaprolu Please fix the color of `Advanced search` as well. It is not matching Google design.

**Comment by nikhilrayaprolu** [comment]
@harshit98 squashed :smile:

**Comment by harshit98** [comment]
@nikhilrayaprolu Please go through, other suggestions as well.

**Comment by nikhilrayaprolu** [comment]
@harshit98  done with the changes

**Comment by nikhilrayaprolu** [comment]
@harshit98  done with the changes.

File: src/app/footer-navbar/footer-navbar.component.css [hunk 1]
```diff
@@ -8,8 +8,6 @@
   justify-content: space-between;
   align-items: center;
   margin-bottom: 0px;
-  position: fixed;
-  bottom: 0px;
```
Comment (harshit98) [inline src/app/footer-navbar/footer-navbar.component.css]: @nikhilrayaprolu why you removed those lines? They fixed the issue #244

Comment (nikhilrayaprolu) [inline src/app/footer-navbar/footer-navbar.component.css]: The position attributes should be independent of the child component I inherit, they should depend only on parent component. when I use the same component in other parent components, I should not have any internal css factors of the child component affecting the position in the parent component, there should be independency to apply my own css  in each parent component I inherit this. so where ever you want to fix the component as fixed , write this css only in the parent component but not in the child component, such that we could have other positions defined in other parent components.

**Pull request merged**